roles/test_operator/tasks/tempest-tests.yml

```diff
@@ -32,13 +32,20 @@
       register:
         list_allowed
 
+    - name: Prepare includeList for Tempest CR
+      set_fact:
```

Comment:
Please, use FQN module names: `ansible.builtin.set_fact`. It's encourage by Ansible guidelines and our linting.